controllers/configmap_controller.go

```diff
@@ -343,7 +343,8 @@ func hasAssociatedInstance(nodeAddresses []core.NodeAddress, instances []*instan
 		if parseAddr := net.ParseIP(nodeAddress.Address); parseAddr != nil {
 			dnsAddresses, err := net.LookupAddr(nodeAddress.Address)
 			if err != nil {
-				return false, errors.Wrapf(err, "failed to lookup DNS for IP %s", nodeAddress.Address)
+				// skip node's address without reverse lookup records
+				continue
```

Comment:
just realized that makes it so no return error is needed for this function signature

Reply:
Oh! I missed that 😞 . Fixed in [923f7f9 ](https://github.com/openshift/windows-machine-config-operator/commit/923f7f99fa3712830457fa80cdc3b1bd823044aa)